pkg/model/volume.go

```diff
@@ -107,6 +107,9 @@ type VolumeAttachmentSpec struct {
 
 	// See details in `ConnectionInfo`
 	ConnectionInfo `json:"connectionInfo,omitempty"`
+
+	// The protocl
+	Protocol string `json:"protocol,omitempty"`
```

Comment:
To me, it's unacceptable for users to specify access protocol when creating volume attachment.

Reply:
Protocol here is provided by pool.